### PR TITLE
Added tests for converted notebook preprocessor

### PR DESCRIPTION
**Pull request by karthikv2k**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/fairing/226)

### CODE REVIEW

File: tests/unit/preprocessors/test_converted_notebook_preprocessor.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    files = preprocessor.preprocess()
+    assert Path(CONVERTED_NOTEBOOK_PATH) in files
+
+def test_get_command():
+    preprocessor = ConvertNotebookPreprocessor(notebook_file=NOTEBOOK_PATH)
+    preprocessor.preprocess()
+    command = preprocessor.get_command()
+    conv_notebook_context_path = posixpath.join(DEFAULT_DEST_PREFIX, CONVERTED_NOTEBOOK_PATH)
+    expected_command = 'python {}'.format(conv_notebook_context_path)
+    print(command)
```
Comment (abhi-g) [inline tests/unit/preprocessors/test_converted_notebook_preprocessor.py]: Can we add a prefix to this print value such as "Command: "

File: tests/unit/preprocessors/test_converted_notebook_preprocessor.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+NOTEBOOK_PATH = os.path.relpath(
+    os.path.join(os.path.dirname(__file__), 'test_notebook.ipynb'))
+CONVERTED_NOTEBOOK_PATH = NOTEBOOK_PATH.replace(".ipynb",".py")
+
+def test_preprocess():
+    preprocessor = ConvertNotebookPreprocessor(notebook_file=NOTEBOOK_PATH)
+    files = preprocessor.preprocess()
+    assert Path(CONVERTED_NOTEBOOK_PATH) in files
+
+def test_get_command():
```
Comment (abhi-g) [inline tests/unit/preprocessors/test_converted_notebook_preprocessor.py]: Does pytest execute the tests in order? i.e. when executing this one, does it guarantee that previous tests passed or were at least ran? that way if test_preprocess fails, we can ignore errors from this test.

Comment (karthikv2k) [reply tests/unit/preprocessors/test_converted_notebook_preprocessor.py]: No. It shouldn't by design. I think what you are asking for is a smoke test but these are unit tests so we wont incur time running all the tests so no gating mechanism is necessary. However, we can do something for integration tests where unit or some other tests failures would not run the remaining tests.

File: tests/unit/preprocessors/test_converted_notebook_preprocessor.py (hunk 1 above)
Comment (karthikv2k) [reply tests/unit/preprocessors/test_converted_notebook_preprocessor.py]: it was a debug statement so removed it.

**Pull request merged**